addons/mail/static/src/core/common/chat_window_service.js

```diff
@@ -34,10 +34,10 @@ export class ChatWindowService {
             thread,
             replaceNewMessageChatWindow,
         });
-        chatWindow.autofocus++;
         if (thread) {
             thread.state = "open";
         }
+        chatWindow.autofocus++;
```

Comment:
@alexkuhn can you have a look at this? I had to move it down because the change of `thread.state` has the side effect of resetting the focus (due to the `onChange` in the thread model)

Reply:
@tsm-odoo

Reply:
Should remove diff here, and make sure the onChange only changes autofocus if the chat window didn't already exist.